lib/src/l18n/localization.dart

```diff
@@ -29,9 +29,15 @@ class FeedbackLocalization extends StatelessWidget {
     if (delegates != null) {
       mergedDelegates.insertAll(0, delegates!);
     }
+
+    Locale fallbackLocale = const Locale('en', 'US');
+    if(window.locale != null && window.locale.toString() != '') {
```

Comment:
Why do we need to check for `window.locale.toString() != ''`?

Reply:
It throws warnings when checking for window.locale != null, as it is of type Locale and theoretically cannot be null. In reality, however, it can be null. This is why the warning gets suppressed and I added an extra check to catch a case where it might be not null, but not filled out (I can imagine that this can happen in the web environment). Long story short - fallback for potential client inconsistencies.

Reply:
I just looked that up. `locale.toString()` never evaluates to an empty string because you can't initialize it without a language code, see https://api.flutter.dev/flutter/package-intl_locale/Locale-class.html

Reply:
https://api.flutter.dev/flutter/dart-ui/SingletonFlutterWindow/locale.html is as you said never null. I guess we'll have to ignore the lint for now and always do a null check.